sdk/elixir/runtime/dagger.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "elixir-sdk",
+  "sdk": "go",
+  "source": ".",
+  "engineVersion": "v0.11.2"
```

Comment:
Is it ok to use this engine version? I can't find what version we should use  but it's always get generated by `dagger develop`.

Reply:
Yeah, it's ok.